pkg/plugin/twinmaker/handler.go

```diff
@@ -623,25 +623,18 @@ func (s *twinMakerHandler) GetWriteSessionToken(ctx context.Context, duration ti
 	info := models.TokenInfo{}
 	credentials, err := s.client.GetWriteSessionToken(ctx, duration, workspaceId)
 	if err != nil {
-		HandleGetTokenError(err)
-		return info, err
+		return info, HandleGetTokenError(err)
 	}
 	return SetInfo(credentials, info), err
 }
 
-func HandleGetTokenError(err error) {
+func HandleGetTokenError(err error) error {
 	if aerr, ok := err.(awserr.Error); ok {
-		switch aerr.Code() {
-		case sts.ErrCodeRegionDisabledException:
-			fmt.Println(sts.ErrCodeRegionDisabledException, aerr.Error())
-		default:
-			fmt.Println(aerr.Error())
-		}
-	} else {
-		// Print the error, cast err to awserr.Error to get the Code and
-		// Message from an error.
-		fmt.Println(err.Error())
+		log.DefaultLogger.Error("error getting session token", "code", aerr.Code(), "error", aerr)
```

Comment:
@haweston i updated this to use the grafana plugin SDK logger. it seems like having the code might be useful info in all cases where the info is available, so i added a `code` field

Reply:
Okay, yeah that's definitely better.